addOns/retire/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Updated with upstream retire.js pattern changes.
 - Depends on an updated version of the Common Library add-on.
+- Maintenance changes (Issue 8940).
```

Comment:
This is simply maintenance changes, that issue does not apply to this rule.

Reply:
Okay I'll remove it, I just figured since retire was listed and discussed I may as well reference it.

Reply:
It's already being referenced in the PR/issue.

Reply:
Done